web/app.js

```diff
@@ -3100,6 +3100,7 @@ function webViewerKeyDown(evt) {
     curElementTagName === "INPUT" ||
     curElementTagName === "TEXTAREA" ||
     curElementTagName === "SELECT" ||
+    curElementTagName === "BUTTON" ||
```

Comment:
Could we "fix" patch by changing this as follows?
```suggestion
    (curElementTagName === "BUTTON" &&
      (evt.keyCode === /* Enter = */ 13 || evt.keyCode === /* Space = */ 32)) ||
```